include/openthread/srp_server.h

```diff
@@ -583,6 +657,45 @@ const uint8_t *otSrpServerServiceGetTxtData(const otSrpServerService *aService,
  */
 const otSrpServerHost *otSrpServerServiceGetHost(const otSrpServerService *aService);
 
+/**
+ * This function returns the lease time in seconds of a given service.
+ *
+ * @param[in]  aService  A pointer to the SRP server service.
+ *
+ * @returns  An integer that represents the lease time in seconds of the given service.
+ *
+ */
+uint32_t otSrpServerServiceGetLease(const otSrpServerService *aService);
```

Comment:
nit: Can we get Lease, KeyLease, RemainingLease, RemainingKeyLease in one API, e.g. `otSrpServerServiceGetLeaseInfo` ?

not sure which one is better though. 
Any preference? @abtink @wgtdkp @superwhd

Reply:
SGTM. The same change can be applied to `otSrpServerHost*`

Reply:
Either way is fine. I agree having fewer APIs may be better.